assets/blocks/cash-app-pay/utils.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * External dependencies
+ */
+import { getSetting } from '@woocommerce/settings';
+import { dispatch } from '@wordpress/data';
+
+
+const { PAYMENT_STORE_KEY } = window.wc.wcBlocksData;
+import { PAYMENT_METHOD_ID } from './constants';
+let cachedSquareCashAppData = null;
+
+/**
+ * Square settings that comes from the server
+ *
+ * @return {SquareServerData} Square server data.
+ */
+export const getSquareCashAppPayServerData = () => {
+	if (cachedSquareCashAppData !== null) {
+		return cachedSquareCashAppData;
+	}
+
+	const squareData = getSetting('square_cash_app_pay_data', null);
+
+	if (!squareData) {
+		throw new Error(
+			'Square Cash App Pay initialization data is not available'
```

Comment:
@jeffpaul, this will be displayed in the browser console log and not directly to the merchant/seller. Also, this is more likely to happen due to a runtime error in the code rather than an incorrect configuration.